uwreact_bringup/launch/zeds.launch

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0"?>
+
+<launch>
+    <!-- TODO(wraftus): Make names of ZEDs more descriptive depending on location on robot-->
+    <arg name="zed_id1"               default="0" />
+    <arg name="verbose"               default="false" />
+
+    <arg name="resolution"            default="2" /> 
+    <!--0=RESOLUTION_HD2K, 1=RESOLUTION_HD1080, 2=RESOLUTION_HD720, 3=RESOLUTION_VGA -->
+    <arg name="frame_rate"            default="30" />
+    <!-- RESOLUTION_HD2K   -> 2208*1242, available framerates: 15 fps.
+         RESOLUTION_HD1080 -> 1920*1080, available framerates: 15, 30 fps.
+         RESOLUTION_HD720  -> 1280*720,  available framerates: 15, 30, 60 fps.
+         RESOLUTION_VGA    -> 672*376,   available framerates: 15, 30, 60, 100 fps. -->
+
+    <!-- Definition coordinate frames -->
+    <arg name="publish_tf"                    default="true" />
+    <arg name="publish_map_tf"                default="true" />
+    <arg name="base_frame"                    default="zed_camera_center" />
+
+    <arg name="camera_flip"                   default="false" />
+
+    <arg name="depth"                         default="depth/depth_registered" />
+    <arg name="depth_cam_info"                default="depth/camera_info" />
+    <arg name="laserscan"                     default="depth/laserscan"/>
```

Comment:
These topics should not prepend `depth/<whatever>`, since they're already going to be prepended with the namespace from the `<group>`. `zed1/laserscan` is better than `zed1/depth/laserscan`

Reply:
Okay, I'll do it for depth and laserscan, but I think we should keep it as depth/camera_info since the zed wrapper publishes rgb/camera_info, left/camera_info, etc. to avoid confusion.

Reply:
Gotcha, I didn't realize that. Sounds good :+1: